docs/tutorials/steps_to_train_models_with_multi_gpus_using_dp.md

```diff
@@ -0,0 +1,57 @@
+# Steps to train models with multiple gpus using DataParallel
+
+If you are interested in training deep forecasting models in parallel using multiple GPUs, please follow the instructions below.
+
+**Notes**: Currently, we only support **DataParallel**, and it is supported only on the **Sequential Backend**.
```

Comment:
We have experimented DataParallel with ray backend, and it seems to work too. Please verify this and update this note if it works with ray backend.

Reply:
I have tested on my platform and it works well using Ray Backend. I have updated the note in the document.